docs/specifications/README.md

```diff
@@ -3,9 +3,12 @@
 
 ### Purpose of this directory
 
-This contents of this directory includes detailed specifications for all the
-DNS tests validated by the "zonemaster-engine".
+This contents of this directory includes detailed technical specifications for all the
```

Comment:
"This contents of this directory includes" -> "This directory contains the"?

Reply:
Ok.